src/dbus/server/introspect.xml

```diff
@@ -740,44 +740,44 @@
     <literallayout>
         struct
         {
-          uint64_t  mId;                // The unique id for a mapping session.
+          uint64_t  mId;              // The unique id for a mapping session.
           uint8_t[] mIp4;             // The IPv4 address of the mapping.
           uint8_t[] mIp6;             // The IPv6 address of the mapping.
-          uint32_t  mRemainingTimeMs;   // Remaining time before expiry in milliseconds.
+          uint32_t  mRemainingTimeMs; // Remaining time before expiry in milliseconds.
 
```

Comment:
Seems we have to remote these empty lines or it won't generate correct XML

Reply:
Removed the empty lines.